app/controllers/profiles_controller.rb

```diff
@@ -5,9 +5,9 @@ class ProfilesController < ApplicationController
   include CategoriesHelper
   include ActsAsTaggableOn::TagsHelper
   include SearchHelper
+  include Pagy::Backend
```

Comment:
why did you add this inclusion? Was it missing before or did you change anything else with pagy which I did not see?

Reply:
with this I can use the pagy method: _pagy_array_